mne/epochs.py

```diff
@@ -735,7 +735,7 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         # pick channels
         if picks is None:
             picks = pick_types(evoked.info, meg=True, eeg=True, ref_meg=True,
-                               stim=False, eog=False, ecg=False,
+                               stim=False, eog=False, ecg=False, seeg=True,
                                emg=False, exclude=[])
```

Comment:
Actually this is almost the default behavior of ``pick_types``. The convenience function would basically be ``pick_types(info, eeg=True, seeg=True)``. I was thinking of factoring by just changing the calls to that, but I feel it is less readable.